kafkacat/Dockerfile.ubi8

```diff
@@ -24,18 +24,18 @@ FROM ${DOCKER_UPSTREAM_REGISTRY}confluentinc/cp-base-new:${DOCKER_UPSTREAM_TAG}
 WORKDIR /build
 
 ENV VERSION=1.5.0-1
-ENV BUILD_PACKAGES="which git make cmake gcc-c++ zlib-devel curl curl-devel openssl-devel openldap-devel cyrus-sasl-devel pkgconfig lz4-devel libzstd-devel" 
+ENV BUILD_PACKAGES="which git make cmake gcc-c++ zlib-devel curl curl-devel openssl-devel openldap-devel cyrus-sasl-devel pkgconfig lz4-devel wget tar"
 
 USER root
 
 RUN echo "Building kafkacat ....." \
     && dnf install -y https://dl.fedoraproject.org/pub/epel/epel-release-latest-8.noarch.rpm \
-    && yum -q -y update \
-    && yum -q install -y $BUILD_PACKAGES \
-    && yum clean all \
-    && git clone https://github.com/edenhill/kafkacat \
+    && dnf -y update \
+    && dnf install -y $BUILD_PACKAGES \
+    && dnf clean all \
+    && git clone https://github.com/andrewegel/kafkacat \
     && cd kafkacat \
-    && git checkout tags/debian/$VERSION \
+    && git checkout origin/master \
```

Comment:
and keep this for now (we'll eventually use a tag).
Not sure why this Dockerfile used debian's tag.